discord/ext/pages/pagination.py

```diff
@@ -254,6 +254,8 @@ class PageGroup:
         Whether the buttons get disabled when the paginator view times out.
     use_default_buttons: :class:`bool`
         Whether to use the default buttons (i.e. ``first``, ``prev``, ``page_indicator``, ``next``, ``last``)
+    use_styled_buttons: :class:`bool` .. versionadded:: 2.4
+        The same as ``use_default_buttons`` but uses emojis for labels, overwrites ``use_default_buttons`` when used.
```

Comment:
It should look something more like this :)
```suggestion
    use_styled_buttons: :class:`bool`
        The same as ``use_default_buttons`` but uses emojis for labels, overwrites ``use_default_buttons`` when used.
        
        .. versionadded:: 2.4
```

Reply:
mb :/